### PR TITLE
ci: use Athens GOPROXY in module mode

### PR DESCRIPTION
**Pull request by sbinet**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ matrix:
        - TAGS="-tags travis"
        - COVERAGE="-race"
        - GO111MODULE="on"
+       - GOPROXY="https://athens.azurefd.net"
```
Comment (jaybatra26) [inline .travis.yml]: @sbinet, Why do we need a proxy server in travis CI?

Comment (sbinet) [reply .travis.yml]: This isn't for Travis only, per se, but to test modules and the expected new package delivery system:
https://blog.golang.org/modules2019

Comment (jaybatra26) [reply .travis.yml]: okay, Thanks. So, are these mirrors paid? Also, I just learned that these could be a victim to man in middle attack. What are your thoughts on this?

Comment (sbinet) [reply .travis.yml]: athens is supposed to be a mirror operated by the community.
there will be bills to be paid, for sure, but I think the gophers won't have to give a credit card upfront.

https should address most of the MiM attacks (all?).
I am not a security expert, though.

**Comment by sbinet** [comment]
superceded by https://github.com/go-hep/hep/pull/487

**Pull request closed without merging**